src/Cooklang.l

```diff
@@ -21,273 +20,21 @@ PC_5          \xef\xbc\xbd|\xef\xbc\xbf|\xef\xbd\x9b|\xef\xbd\x9d|\xef\xbd\x9f|\
-AC_75          \xef\xbb\x8b|\xef\xbb\x8c|\xef\xbb\x8d|\xef\xbb\x8e|\xef\xbb\x8f|\xef\xbb\x90|\xef\xbb\x91|\xef\xbb\x92|\xef\xbb\x93|\xef\xbb\x94|\xef\xbb\x95|\xef\xbb\x96|\xef\xbb\x97|\xef\xbb\x98|\xef\xbb\x99|\xef\xbb\x9a|\xef\xbb\x9b|\xef\xbb\x9c|\xef\xbb\x9d|\xef\xbb\x9e|\xef\xbb\x9f|\xef\xbb\xa0|\xef\xbb\xa1|\xef\xbb\xa2|\xef\xbb\xa3|\xef\xbb\xa4|\xef\xbb\xa5|\xef\xbb\xa6|\xef\xbb\xa7|\xef\xbb\xa8|\xef\xbb\xa9|\xef\xbb\xaa|\xef\xbb\xab|\xef\xbb\xac|\xef\xbb\xad|\xef\xbb\xae|\xef\xbb\xaf|\xef\xbb\xb0|\xef\xbb\xb1|\xef\xbb\xb2|\xef\xbb\xb3|\xef\xbb\xb4|\xef\xbb\xb5|\xef\xbb\xb6|\xef\xbb\xb7|\xef\xbb\xb8|\xef\xbb\xb9|\xef\xbb\xba|\xef\xbb\xbb|\xef\xbb\xbc|\xef\xbc\xa1|\xef\xbc\xa2|\xef\xbc\xa3|\xef\xbc\xa4|\xef\xbc\xa5|\xef\xbc\xa6|\xef\xbc\xa7|\xef\xbc\xa8|\xef\xbc\xa9|\xef\xbc\xaa|\xef\xbc\xab|\xef\xbc\xac|\xef\xbc\xad|\xef\xbc\xae|\xef\xbc\xaf|\xef\xbc\xb0|\xef\xbc\xb1|\xef\xbc\xb2|\xef\xbc\xb3|\xef\xbc\xb4|\xef\xbc\xb5|\xef\xbc\xb6|\xef\xbc\xb7|\xef\xbc\xb8|\xef\xbc\xb9|\xef\xbc\xba|\xef\xbd\x81|\xef\xbd\x82|\xef\xbd\x83|\xef\xbd\x84|\xef\xbd\x85|\xef\xbd\x86|\xef\xbd\x87|\xef\xbd\x88|\xef\xbd\x89|\xef\xbd\x8a|\xef\xbd\x8b|\xef\xbd\x8c|\xef\xbd\x8d|\xef\xbd\x8e|\xef\xbd\x8f|\xef\xbd\x90|\xef\xbd\x91|\xef\xbd\x92|\xef\xbd\x93|\xef\xbd\x94|\xef\xbd\x95|\xef\xbd\x96|\xef\xbd\x97|\xef\xbd\x98|\xef\xbd\x99|\xef\xbd\x9a|\xef\xbd\xa6|\xef\xbd\xa7|\xef\xbd\xa8|\xef\xbd\xa9|\xef\xbd\xaa|\xef\xbd\xab|\xef\xbd\xac|\xef\xbd\xad|\xef\xbd\xae|\xef\xbd\xaf|\xef\xbd\xb0|\xef\xbd\xb1|\xef\xbd\xb2|\xef\xbd\xb3|\xef\xbd\xb4|\xef\xbd\xb5|
```

Comment:
We don't need number anymore?

Reply:
indeed.
The reason is that in the lexer, we parse differently the number depending if it is a DIGIT/INTEGER/DECIMAL (basic `strtod`) or if it is a FRACTIONAL (need to split the string on the "/" and compute the division)

but then it is a NUMBER symbol that is sent to Bison, just not this NUMBER ^^'